src/lib/tvgSceneImpl.h

```diff
@@ -59,45 +66,50 @@ struct Scene::Impl
 
     bool render(RenderMethod &renderer)
     {
-        //TODO: composition begin
-        //auto data = renderer.beginComp();
+        void* ctx = nullptr;
+
+        //Half translucent. This requires intermediate composition.
+        if (opacity < 255 && opacity > 0) {
+            float x, y, w, h;
+            bounds(&x, &y, &w, &h);
+            ctx = renderer.beginComposite(roundf(x), roundf(y), roundf(w), roundf(h));
```

Comment:
The parameter of beginComposite is uint32_t.
If x and y are negative, i think it may cause problems during the typecasting process.